tests/test_linearity.py

```diff
@@ -85,6 +87,9 @@ def _create_ptc(self, amp_names, exp_times, means, ccobcurr=None):
         """
         exp_id_pairs = np.arange(len(exp_times)*2).reshape((len(exp_times), 2)).tolist()
 
+        if photo_charges is None:
+            photo_charges = np.zeros(len(exp_times)) + np.nan
```

Comment:
Is this more efficient than `np.full(len(exp_times), np.nan)`?

Reply:
Oddly enough, for small N the zeros is faster (but negligible time), for medium N full is faster, and for large N they're very close.  I don't think it matters much, I can change to full if you like.

Reply:
(small ~10; medium ~10000; large ~1000000)

Reply:
I assumed they were both extremely optimized, I think I just find `full` easier to read.